components/form-check/src/FormCheck.js

```diff
@@ -106,29 +117,34 @@ export const FormCheck = ({
 			allChildren.push(
 				<Option
 					key={index}
+					index={index}
 					value={props.value}
 					checked={props.checked || checked.includes(props.value)}
 					handleChange={handleChange}
 					type={type}
 					name={name}
+					name={name}
```

Comment:
Remove duplication

Reply:
Hmm, that was weird. Failed auto merge? 
Fixed